### PR TITLE
fix nightly wheel commands

### PR DESCRIPTION
**Pull request by soumith**
Fixes https://github.com/pytorch/pytorch/issues/23218

### CODE REVIEW

File: assets/quick-start-module.js [hunk 1]
```diff
@@ -519,79 +522,79 @@ function commandMessage(key) {
...
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html <br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
+      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html <br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
 
     "preview,pip,macos,cudanone,python3.7":
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html",
+      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html",
 
     "preview,pip,linux,cudanone,python2.7":
-      "pip install numpy<br />pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch.html",
+      "pip install numpy<br />pip install torch==" + torch_nightly_ver + "+cpu torchvision==" + vision_nightly_ver + "+cpu -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html",
```
Comment (ezyang) [inline assets/quick-start-module.js]: You don't need to hardcode the versions here

**Comment by ezyang** [comment]
We need to make some more updates:

1. In general, instead of specifying version bounds as `torch==1.2.0` we need to stay `torch===1.2.0` to ensure that we don't pick up the local version specifier (as per discussion at https://github.com/pytorch/pytorch/issues/23656#issuecomment-523378231 )

2. We don't need to compute the date for our users; if we pip install torch from the correct channel it will automatically pick up the latest one

File: assets/quick-start-module.js (hunk 1 above)
Comment (peterjc123) [reply assets/quick-start-module.js]: I tried not to hardcode the version during installation but without success.
Log:
```
$>pip install torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple
Looking in links: https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
Collecting torch
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/f8/02/880b468bd382dc79896eaecbeb8ce95e9c4b99a24902874a2cef0b562cea/torch-0.1.2.post2.tar.gz (128kB)
     |████████████████████████████████| 133kB 652kB/s
ERROR: Operation cancelled by user
```

Comment (peterjc123) [reply assets/quick-start-module.js]: Detailed log: https://gist.github.com/peterjc123/ae2e753bdd5628b62ad71984533a9584

Comment (ezyang) [reply assets/quick-start-module.js]: I think the fix is, you also need to pass `--pre` so pip considers preinstall versions too (I don't know if the mirror is involved with the trouble)

Comment (peterjc123) [reply assets/quick-start-module.js]: Yes, it works. Thanks!

**Pull request merged**